src/tedpy/ted6000.py

```diff
@@ -29,19 +30,31 @@ def __init__(self, host: str, async_client: httpx.AsyncClient = None) -> None:
 
         self.endpoint_settings_results: Any = None
         self.endpoint_dashboard_results: Any = None
+        self.endpoint_mtu1_dashboard_results: Any = None
+        self.endpoint_mtu2_dashboard_results: Any = None
+        self.endpoint_mtu3_dashboard_results: Any = None
+        self.endpoint_mtu4_dashboard_results: Any = None
```

Comment:
I don't like how this variable is repeated 4 times, even though there can technically only be up to 4 MTUs.

I think it might be better to rework the update endpoint function so that `self.endpoint_mtu_dashboard_results` is initialized to a `dict()`, then when the second `param` argument is not `None`, `self.endpoint_mtu_dashboard_results[param]` gets set.

Reply:
My lack of Python knowledge is the limiting factor here. The issue is that `_update_endpoint` uses `setattr(self, attr, xmltodict.parse(response.text))` so I couldn't figure out how to combine that data. I asked on the Python discord and confused the crap out of everyone lol!

Open to suggestions on how to Pythonify it. The map/lambda stuff is a bit of witchcraft I somehow go to work as well.

Reply:
I was imagining something like this:

```python
async def _update_endpoint(self, attr: str, url: str, param: str = None) -> None:
    formatted_url = url.format(self.host, param)
    response = await self._async_fetch_with_retry(formatted_url)
    try:
        if param is None:
            # write to self[attr]
            setattr(self, attr, xmltodict.parse(response.text))
        else:
            # write to self[attr][param]; assume self[attr] was initialized to dict()
            getattr(self, attr)[param] = xmltodict.parse(response.text)
    except xmltodict.expat.ExpatError:
        raise
```

And then the variable is initialized as:


```python
self.endpoint_mtu1_dashboard_results: Dict[int, Any] = dict()
```

Reply:
I believe ebf939f should resolve this comment. Very slick, thanks!

Reply:
My only concern is I cheated a little bit by setting position[0] to be the TOTALs and [1-4] to be the individual MTUs. It doesn't leave much room for LOAD and GEN expansion but then again we weren't going to target those yet anyway because we don't know what they do.

Reply:
Looks great!

When we get to that point there can be 3 of these dictionaries for load, gen, and net.